site/frontend/src/utils/navigation.ts

```diff
@@ -2,12 +2,12 @@ export function createUrlParams(params: Dict<string>): URLSearchParams {
     return createUrlWithParams(params).searchParams;
 }
 
-export function createUrlWithParams(params: Dict<string>): URL {
+export function createUrlWithParams(params: Dict<any>): URL {
     const originalUrl = window.location.toString();
     const url = new URL(originalUrl);
     for (const [key, value] of Object.entries(params)) {
         if (value !== null && value !== undefined && value !== "") {
```

Comment:
if values aren't strings anymore then should this check whether `toString()` returns an empty string instead ?

Reply:
Good point!